valve-fsm.txt

```diff
@@ -83,8 +126,8 @@ int main(void) {
         }
 
         state = valve_fsm(state, event, timer_expired);
-    }
+	ind_lights(valve_state)
+    };
```

Comment:
no ; needed at the end of function definitions

Reply:
done thanks